### PR TITLE
Kapacitor authorization and authentication

### LINKED ISSUE #2690: Kapacitor authentication and authorization
Kapacitor 1.6 ports the user authentication and authorization functionality into Kapacitor Enterprise into OSS. We need to document how to use this functionality.

I think we can just port the following docs into the new docs:

- https://github.com/influxdata/docs.influxdata.com-ARCHIVE/blob/master/content/enterprise_kapacitor/v1.5/administration/auth.md
- https://github.com/influxdata/docs.influxdata.com-ARCHIVE/blob/master/content/enterprise_kapacitor/v1.5/administration/security.md

This content needs to be reviewed and updated to match the style of v2 documentation.

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2690 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/kapacitor/v1.6/user-types-permissions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Kapacitor user types and permissions
+description: >
+  View Kapaictor user types and permissions available when using internal
```
Comment (kelseiv) [inline content/kapacitor/v1.6/user-types-permissions.md:4]:
```suggestion
  View Kapacitor user types and permissions available when using internal
```

File: content/kapacitor/v1.6/administration/auth/_index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+##### Example authentication settings in the kapacitor.conf
+```toml
+[http]
+  # ...
+  auth-enabled = true
+  # ...
+
+[auth]
+  # Enable authentication for Kapacitor
```
Comment (kelseiv) [inline content/kapacitor/v1.6/administration/auth/_index.md:53]:
```suggestion
  # Enable authentication for Kapacitor.
```

Comment (kelseiv) [reply content/kapacitor/v1.6/administration/auth/_index.md:53]: Or all comments w/o

File: content/kapacitor/v1.6/administration/auth/influxdb-enterprise-auth.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
+---
+title: Use InfluxDB Enterprise authorizations
+seotitle: Authenticate with Kapacitor using InfluxDB Enterprise
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/kapacitor/v1.6/administration/auth/influxdb-enterprise-auth.md:3]: This is too long of an SEO title. The greatest weight is placed on the first 4 words of a page title.

File: content/kapacitor/v1.6/administration/auth/influxdb-enterprise-auth.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+- [Enable and configure Kapacitor authentication](#enable-and-configure-kapacitor-authentication)
+- [Authenticate with Kapacitor](#authenticate-with-kapacitor)
+
+## How authentication works
+The process of using InfluxDB Enterprise authorizations to authenticate with Kapacitor
+involves three components of the enterprise [TICK stack](/platform/#influxdata-1x-tick-stack):
+
+- [InfluxDB Enterprise](/{{< latest "enterprise_influxdb" >}}/) meta nodes
+- [Kapacitor](/kapacitor/v1.6/)
+- [Chronograf](/{{< latest "chronograf" >}}/) _(to aid in the creation of users and roles)_
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/kapacitor/v1.6/administration/auth/influxdb-enterprise-auth.md:28]: Yes, it is optional, but not covered in this doc. It is linked to later.

File: content/kapacitor/v1.6/administration/auth/influxdb-enterprise-auth.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+weight: 202
+---
+
+Use user-based authorizations stored and managed in **InfluxDB Enterprise** to
+authenticate requests to the Kapacitor HTTP API.
+
+- [How authentication works](#how-authentication-works)
+- [Create an InfluxDB Enterprise user or role with Kapacitor permissions](#create-an-influxdb-enterprise-user-or-role-with-kapacitor-permissions)
+- [Enable and configure Kapacitor authentication](#enable-and-configure-kapacitor-authentication)
+- [Authenticate with Kapacitor](#authenticate-with-kapacitor)
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/kapacitor/v1.6/administration/auth/influxdb-enterprise-auth.md:20]: I thought about pulling this content into it's own page, but it's only applicable to this specific use case (using InfluxDB Ent auths to auth with Kapacitor). That's why I left it here.

File: content/kapacitor/v1.6/administration/auth/kapacitor-auth.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
+---
+title: Use internal Kapacitor authorizations
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/kapacitor/v1.6/administration/auth/kapacitor-auth.md]: Good call. I can do that.

**Review by kelseiv — APPROVED** [review]

**Pull request merged**